.github/workflows/dotnet-desktop.yml

```
@@ -0,0 +1,115 @@
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+
+    strategy:
+      matrix:
+        configuration: [Debug, Release]
+
+    runs-on: windows-latest  # For a list of available runner types, refer to
+                             # https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)